borges/templates/_helpers.tpl

```diff
@@ -14,3 +14,10 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Create database connection string
+*/}}
+{{- define "database" -}}
+postgres://{{ .user }}:{{ .pass }}@{{ .host }}/{{ .name }}?application_name=borges&sslmode=disable&connect_timeout=30
```

Comment:
Why do you want to keep configuration as it was when now postgres in borges has a proper connection string? This definitely looks cool but it is much more flexible to be able to pass directly the connection string instead the user/pass/host/name that will be used to build the connection string. It would also make all the dependencies configuration the same

Reply:
Changed